tests/e2e/framework/kubernetes.go

```diff
@@ -55,6 +56,7 @@ const (
 	mcAuthInstallFileNamespace     = "istio-auth-multicluster.yaml"
 	mcRemoteInstallFile            = "istio-remote.yaml"
 	mcSplitHorizonInstallFile      = "istio-multicluster-split-horizon.yaml"
+	authOperatorInstallFile        = "istio-operator.yaml"
```

Comment:
Where is this file?

Reply:
this file is created by the e2e.sh script in https://github.com/istio/operator/pull/379

Cheers
-steve